pom.xml

```diff
@@ -47,10 +53,10 @@
 	    	mvn clean verify
 
 		* Change project's version:
-			mvn org.eclipse.tycho:tycho-versions-plugin:set-version -DnewVersion=major.minor.bug
+			mvn org.eclipse.tycho:tycho-versions-plugin:set-version -D newVersion=major.minor.bug -D skipTests
```

Comment:
I think there should be no space between `-D` and the property name

Reply:
From my experience it works just as well with a space. I'll remove it because even the official documentation shows no space but do you know the rationale behind this? I haven't been able to find a clear explanation.

Reply:
Indeed I've always seen no space after `-D`